content/docs/welcome-guide/setup/setup-from-github.md

```diff
@@ -202,23 +253,46 @@ Now that you have a local copy of the O3DE source, you can build the engine, inc
     cmake --build build/windows_vs2019 --target INSTALL --config profile -- /m
     ```
 
-    {{< note >}}
-The `/m` is a recommended build tool optimization, which tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
-    {{< /note >}}
+    The `/m` is a recommended build tool optimization, which tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
 
 The engine will take a while to build. If you've used all the example commands in these steps, when the build is complete, the engine tools and other binaries can be found in `C:\o3de-install\bin\Windows\profile`.
```

Comment:
```suggestion
The engine takes a while to build. If you've used all the example commands in these steps, when the build is complete, you can find the engine tools and other binaries in `C:\o3de-install\bin\Windows\profile`.
```
Forgot to apply this edit to the parallel outro sentence on this tab. (Same edit as the other tab.)

Reply:
Whoops, I should have checked for that too.